src/main/java/nl/workingtalent/wtacademy/review/ReviewService.java

```diff
@@ -1,8 +1,40 @@
 package nl.workingtalent.wtacademy.review;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
 
+import java.util.Optional;
+
 @Service
 public class ReviewService {
 
+    private final int pageSize = 5;
+
```

Comment:
Word niet gebruikt omdat je deze waarde in controller al defined.

Reply:
Aangepast